doc/add-new-publisher.md

```diff
@@ -0,0 +1,9 @@
+# Add a new publisher
```

Comment:
this is cool but I don't know if we are using the doc this way. For now I'm putting everything in the README waiting for a more structured approach. Did you configure some tool locally?